src/test/java/io/vincenzopalazzo/placeholder/DemoFrame.java

```diff
@@ -35,7 +36,8 @@
       UIManager.put("TextFieldPlaceholder.inactiveBackground", MaterialColors.BLACK);
       UIManager.put("TextFieldPlaceholder.selectionBackground", MaterialColors.LIGHT_BLUE_400);
       UIManager.put("TextFieldPlaceholder.selectionForeground", MaterialColors.BLACK);*/
-      UIManager.put("TextFieldPlaceholder.placeholderColor", MaterialColors.RED_800);
+      UIManager.put("Placeholder.foreground", MaterialColors.COSMO_DARK_GRAY);
```

Comment:
@zanderson9 to set the color of the placeholder, foreground and background are supported.